lib/models/timetable/course_model.dart

```diff
@@ -30,6 +37,9 @@ class CourseModel implements Comparable<CourseModel> {
     venue = json['venue'];
     midsem = json['midsem'];
     endsem = json['endsem'];
+    midSemVenue = json['midSemVenue'];
+    endSemVenue = json['endSemVenue'];
```

Comment:
the response fields are midsemVenue and endsemVenue where S is capital
correct these properly

Reply:
done